WorldBank-Transport/DRIVER
# Enable filtering records by creation date

## Overview
Adds a second date range filter parallel to the occurrence filter to enable a user to limit records show to ones that were created inside of DRIVER inside a date range, in addition to filtering by when they happened.

### Checklist


### Demo
<img width="808" alt="screen shot 2018-08-30 at 2 46 50 pm" src="https://user-images.githubusercontent.com/1032849/44872473-a08af080-ac63-11e8-8110-94d8e5aebb09.png">

## Testing Instructions
- Create a record and set the Occurred date in the past (But within the past 3 months)
  - The record should be displayed
- Set the Created Date filter to end after the date the record was set to have occurred, but before present day
  - The record should not be displayed
- Adjust the Created Date filter to end in the future, after the current date
  - The record should be displayed again
- Adjust the Occurred Date filter to begin after the filter was set to have occurred, but before present day
  - The record should not be displayed

Closes #159625879
